layouts/shortcodes/dbm-sqlserver-agent-config-examples.md

```diff
@@ -0,0 +1,126 @@
+### Connecting with DSN using the ODBC driver
+1. Configure the `odbc.ini` file based on your DSN settings.
+
+    Example: 
+    ```text
+    [DATADOG]
```

Comment:
Is this right? Shouldn't it be the name of the driver or does it not matter? 

going based off of this https://docs.datadoghq.com/database_monitoring/setup_sql_server/troubleshooting/#connecting-to-sql-server-on-a-linux-host